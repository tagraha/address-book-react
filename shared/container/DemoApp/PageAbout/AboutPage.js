import React from 'react';
import Helmet from 'react-helmet';

function AboutRoute() {
  return (
    <div style={{ textAlign: 'center' }}>
      <Helmet>
        <title>About</title>
      </Helmet>

      <p>
        Produced with{' '}
        <span role="img" aria-label="heart">
          ❤️
        </span>
        <br />
        <span>Tirta Nugraha (dev.nugrata@gmail.com)</span>
        Github:
        <a
          href="https://github.com/tagraha/address-book-react"
          target="_blank"
          rel="noopener noreferrer" /** https://support.performancefoundry.com/article/186-noopener-noreferrer-on-my-links */
        >
            https://github.com/tagraha/address-book-react
        </a>
      </p>
    </div>
  );
}

export default AboutRoute;
