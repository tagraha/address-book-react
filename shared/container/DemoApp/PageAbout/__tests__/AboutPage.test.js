/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import { shallow } from 'enzyme';

import AboutPage from '../AboutPage';

describe('should render <AboutPage /> page correctly', () => {
  test('renders', () => {
    const wrapper = shallow(<AboutPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
