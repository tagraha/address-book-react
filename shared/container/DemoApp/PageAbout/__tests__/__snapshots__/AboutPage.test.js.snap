// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <AboutPage /> page correctly renders 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      About
    </title>
  </HelmetWrapper>
  <p
    style={
      Object {
        "marginTop": "56px",
      }
    }
  >
    Produced with
     
    <span
      aria-label="heart"
      role="img"
    >
      ❤️
    </span>
    <br />
    <div>
      Tirta Nugraha (dev.nugrata@gmail.com)
    </div>
    <br />
    Github: 
    <a
      href="https://github.com/tagraha/address-book-react"
      rel="noopener noreferrer"
      target="_blank"
    >
      https://github.com/tagraha/address-book-react
    </a>
  </p>
</div>
`;
